plugins/auto-sizes/hooks.php

```diff
@@ -30,7 +30,7 @@ function auto_sizes_update_image_attributes( $attr ) {
 	}
 
 	// Don't add 'auto' to the sizes attribute if it already exists.
-	if ( false !== strpos( $attr['sizes'], 'auto,' ) ) {
+	if ( str_contains( $attr['sizes'], 'auto,' ) ) {
```

Comment:
`str_contains()` is only available as of PHP8. However, since WP 5.9 and later implement a [polyfill](https://github.com/WordPress/wordpress-develop/blob/5afdac7e057bf4faf097079c0704f6e834fdce97/src/wp-includes/compat.php#L455-L475) for this, then this is fine.